### PR TITLE
Schema tests

### PR DESCRIPTION
**Pull request by Errora**
Edit test for schema page

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ coverage_result.txt
 .DS_Store
 .vscode
 luacov.stats.out
+cartridgeWorkspace.code-workspace
```
Comment (rosik) [inline .gitignore]: Is it necessary?

File: CHANGELOG.md [hunk 2]
```diff
@@ -112,6 +112,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Enhance WebUI modals scrolling.
 
+- Success & error notifications in Schema editor.
+
```
Comment (rosik) [inline CHANGELOG.md]: Don't add changes to the section 2.0.0, it's already released.

File: test/cypress/basic_test.lua [hunk 3]
```diff
@@ -122,11 +122,20 @@ function g.test_users()
         ',cypress/integration/remove-user.spec.js' ..
         ',cypress/integration/server-details.spec.js' ..
         ',cypress/integration/login-and-logout.spec.js'..
-        ',cypress/integration/auth-switcher-not-moved.spec.js'
+        ',cypress/integration/auth-switcher-not-moved.spec.js'..
+        ',cypress/integration/schema-editor-with-bootstrap.spec.js'
```
Comment (rosik) [inline test/cypress/basic_test.lua]: Please merge it with `schema-editor.spec.js`

File: test/cypress/basic_test.lua [hunk 4]
```diff
@@ -122,11 +122,20 @@ function g.test_users()
...
     cypress_run('schema-editor.spec.js')
 end
+
+function g.test_uninitialized()
+    local code = os.execute(
+        "cd webui && npx cypress run --config baseUrl='http://localhost:8085' --spec" ..
+        ' cypress/integration/uninitialized.spec.js'
+    )
+    t.assert_equals(code, 0)
+end
```
Comment (rosik) [inline test/cypress/basic_test.lua]: Watch the newlines, please (and in other files too)

File: .gitignore (hunk 1 above)
Comment (Errora) [reply .gitignore]: deleted

File: CHANGELOG.md (hunk 2 above)
Comment (Errora) [reply CHANGELOG.md]: ok

File: test/cypress/basic_test.lua (hunk 3 above)
Comment (Errora) [reply test/cypress/basic_test.lua]: done

**Pull request merged**